src/locales/en_US.js

```diff
@@ -1,12 +1,28 @@
 // @flow
 
 const strings = {
+  accept_button_text: 'Accept',
```

Comment:
Sorting these entries was unnecessary and a waste of time and makes reviewing harder. Please undo and add only necessary strings.